tools/integration_tests/mounting/mount_helper_test.go

```diff
@@ -56,9 +57,15 @@ func (t *MountHelperTest) SetUp(_ *TestInfo) {
 	switch runtime.GOOS {
 	case "darwin":
 		t.helperPath = path.Join(gBuildDir, "sbin/mount_gcsfuse")
```

Comment:
I don't get it. here we already assigning t.helperPath to sbin/mount_gcsfuse. So what is your If condition is doing?

Reply:
The path should not have gBuildDir (temp directory created for testing) if we're going to test on pre-intsalled package. So if the TestInstalledPackage flag is set to true, t.helper variable will be assigned the new path.